js/src/last_week_tonight.js

```diff
@@ -0,0 +1,153 @@
+
+const githubDataFile = path.resolve(__dirname, '../../data/github.yml');
+const githubInfo = yaml.load(fs.readFileSync(githubDataFile));
+const org = githubInfo.org;
+const repos = Object.values(githubInfo.repos);
+
+/* Time period */
+
+const msInWeeks = (weeks) => weeks * 7 * 24 * 60 * 60 * 1e3;
+// End date is always today
+const [endDate] = new Date()
+    .toISOString()
+    .split('T');
+// Start date is one week before today
+const [startDate] = new Date(new Date().getTime() - msInWeeks(1))
+    .toISOString()
+    .split('T');
+
+/* GitHub API */
+
+const octokit = new Octokit({auth: pat});
+const mergedPrsQ = repo =>
+    `repo:${org}/${repo} is:pr is:merged merged:>=${startDate}`;
+const closedIssuesQ = repo =>
+    `repo:${org}/${repo} is:issue is:closed closed:>=${startDate}`;
+
+/* Format issues, PRs and repos as HTML */
+
+/**
+ * Generate the HTML for one closed issues or merged PRs section.
+ * @param {string} title - the the title to use for the section
+ * @param {{html_url: string, number: int, title: string}[]} items - the list of issues/PRs
+ * @returns {string[]} - lines of HTML
+ */
+const getItemsHtml = (title, items) => {
+    if (!items.length) return [];
+
+    return [
+        `<h3>${title}</h3>`,
+        '<ul>',
+        ...items.map(item =>
+            `<li><a href="${item.html_url}">#${item.number}</a>: ${item.title}`,
+        ),
+        '</ul>',
+    ];
+};
+/**
+ * Generate the HTML for the closed issues or merged PRs of one repository.
+ * @param {string} repo - the name of the repository
+ * @param {{html_url: string, number: int, title: string}[]} mergedPrs - the list of PRs
+ * @param {{html_url: string, number: int, title: string}[]} closedIssues - the list of issues
+ * @returns {string[]} - lines of HTML
+ */
+const getRepoHtml = ({repo, mergedPrs, closedIssues}) => {
+    return [
+        `<h2><a href="https://github.com/${org}/${repo}">${repo}</a></h2>`,
+        ...getItemsHtml('Merged PRs', mergedPrs),
+        ...getItemsHtml('Closed issues', closedIssues),
+    ]
+};
+
+// Create post on Make site
+const MAKE_SITE_API = 'https://make.wordpress.org/openverse/wp-json/wp/v2';
+const token = Buffer.from(`${username}:${password}`).toString('base64');
+/**
+ * Post the activities to the Make site.
+ * @param {{
+ *   repo: string,
+ *   mergedPrs: {html_url: string, number: int, title: string}[],
+ *   closedIssues: {html_url: string, number: int, title: string}[]
+ * }[]} activities - the list of repos and their activities
+ * @returns {Promise} - the response for the POST request
+ */
+const postActivities = (activities) => {
+    const report = activities.map(getRepoHtml).join('\n');
+    return fetch(`${MAKE_SITE_API}/posts`, {
+        method: 'POST',
+        headers: {
+            'Content-Type': 'application/json',
+            Authorization: `Basic ${token}`,
+        },
+        body: JSON.stringify({
+            title: `Last week in Openverse: ${startDate} - ${endDate}`,
```

Comment:
Could this follow the existing naming format, just for the sake of consistiency? 
Example: "A Week in Openverse: September 11-17th"

Reply:
If the date format changes, that's fine.

Reply:
I updated the title in ca4cdec, but I'm favouring the ISO-formatted date because the colloquial format requires a considerable amount of edge case handling for ordinal number dates and month/year edges.